.eslintrc

```diff
@@ -2,6 +2,9 @@
     "extends": "girder",
     "rules": {
         "arrow-parens": "error",
+        "eqeqeq": ["error",
+            "always", {"null": "always"}
+        ],
```

Comment:
[It says that we should always use `===`](http://eslint.org/docs/rules/eqeqeq). The options for this rule that we inherit from `eslint-plugin-standard` allow the use of `==` to compare with `null`, so this overrides to not permit that case either.